styles/main.css

```diff
@@ -145,7 +145,28 @@ nav {
 }
 
 .nav-list {
-  display: none;
+  /* display: none; */
+  position: absolute;
+  top: 8vh;
+  left: 0;
+  width: 100vw;
+  height: 92vh;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  gap: 3rem;
+  transition: transform 0.5s ease;
+  transform: translateX(-100%);
```

Comment:
- [ ] Kindly make sure your design matches the original Figna design✌
- Your Design
![screenshot (7)](https://user-images.githubusercontent.com/44624138/145985435-4d08b4d9-7527-410a-a4eb-685481195efc.png)
- Figma Design
![screenshot (8)](https://user-images.githubusercontent.com/44624138/145985499-fc234a39-915d-497c-b41f-d7896c15a615.png)